app/controllers/vehicles_controller.rb

```diff
@@ -8,4 +9,36 @@ def show
     @vehicle = Vehicle.find(params[:id])
```

Comment:
you don't need this one, you should update line 2 to before_action :set_vehicle, only: %i[edit update index]